optimum/exporters/openvino/model_patcher.py

```diff
@@ -1187,12 +1207,14 @@ def __exit__(self, exc_type, exc_value, traceback):
                 layer.self_attn.forward = layer.self_attn._orig_forward
 
 
+# Modified from https://github.com/huggingface/transformers/blob/v4.48.0/src/transformers/models/mpt/modeling_mpt.py#L90
 def _mpt_sdpa_attention_forward(
```

Comment:
I don't think mpt needs this patch 😅
I'll test removing it, along with other undocumented patches, in another PR.